src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const versionOfState = [];
+  let newState = { ...state };
+
+  for (const i of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Let's use more informative names